### PR TITLE
feat(ci): bump VirtualSwitch to 2.6.31 in venom tests and allow TEST_ONLY to trigger CI

### PR DESCRIPTION
**Pull request by JeGoi**
# Description

Two improvements bundled together:

1. **VirtualSwitch 2.6.31 venom test alignment** — updates the VirtualSwitch package version in acceptance tests from `2.6.5-1` to `2.6.31-1`, removes `coa_enabled`/`coa_port` from the deploy config (no
longer part of 2.6.31's schema), and updates `ipfix_destination_port` from `2055` to `4739` (IANA-registered standard introduced in 2.6.11).

2. **`TEST_ONLY` now triggers CI without `test=yes`** — previously `TEST_ONLY=mac_auth` alone would not start any test jobs because the GitLab rules still required `TEST == "yes"` or `test=yes` in the
commit message. All test-related rules now accept `$TEST_ONLY != ""` as a standalone trigger. Also adds support for comma-separated patterns (e.g. `TEST_ONLY=mac_auth,virtualswitch`).

# Impacts

- **CI rules** (`.test_devel_and_maintenance_rules`, `.test_branches_only_rules`, `.publish_ppa_rules`): reviewer should verify the new `|| $TEST_ONLY != ""` conditions don't unintentionally trigger jobs
on scheduled pipelines or pushes without the variable set.
- **`.test_script_job` shell filter**: comma-to-pipe translation (`tr ',' '|'`) before the `grep -qE` — verify edge cases (single value, empty string, trailing comma).
- **Venom test vars**: `ipfix_destination_port` and removal of `coa_enabled`/`coa_port` in `virtualswitch_config_deploy.yml` and `all.yml`.

# Delete branch after merge
YES

# Checklist
- [n/a] Document the feature
- [n/a] Add OpenAPI specification
- [n/a] Add unit tests
- [yes] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* `TEST_ONLY` CI variable now triggers test jobs on its own — no need to also set `TEST=yes` or add `test=yes` to the commit message
* `TEST_ONLY` now accepts comma-separated patterns (e.g. `mac_auth,virtualswitch`)
* Align venom VirtualSwitch acceptance tests with VirtualSwitch 2.6.31 (package bump, config schema update, IANA-standard IPFIX port)

### CODE REVIEW

**Comment by JeGoi** [comment]
https://gitlab.com/inverse-inc/packetfence/-/pipelines/2489366417

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -616,7 +616,10 @@ variables:
           echo "Error: TEST_ONLY contains invalid characters (newline or single quote)."
           exit 1
         fi
-        if echo "$CI_JOB_NAME" | grep -qE "$TEST_ONLY"; then
+        # Accept comma-separated list (e.g. auth,virtual) by translating to regex alternation
+        # Filter out empty items to prevent empty alternation (e.g., "mac_auth," -> "mac_auth|" matches everything)
+        TEST_ONLY_PATTERN=$(printf '%s' "$TEST_ONLY" | tr ',' '\n' | grep -v '^[[:space:]]*$' | tr '\n' '|' | sed 's/|$//')
```
[the remark this answers is not in the document]
Comment (JeGoi) [reply .gitlab-ci.yml:621]: fixed

**Comment by JeGoi** [comment]
https://gitlab.com/inverse-inc/packetfence/-/pipelines/2553421575
